examples/with-langgraph/app/MyRuntimeProvider.tsx

```
@@ -35,8 +41,7 @@ const useMyLangGraphRuntime = () => {
 };
 
 const cloud = new AssistantCloud({
-  baseUrl: "https://api.assistant-ui.com",
-  unstable_projectId: process.env["NEXT_PUBLIC_ASSISTANT_PROJECT_ID"]!,
+  baseUrl: process.env["NEXT_PUBLIC_ASSISTANT_BASE_URL"]!,
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)